test/cms_index.php

```diff
@@ -28,27 +28,154 @@ public function testDisplayAboutMeTextMalform()
         $this->expectException(TypeError::class);
         displayAboutMeText($aboutMeText);
     }
+
     public function testDisplayAboutMeQuoteSuccess()
     {
-        $expected = '<p>test data</p>';
-        $aboutMeText = [['content' => 'test data']];
-        $case = displayAboutMeText($aboutMeText);
+        $expected = '<p class=\'contentEmphasisLine\'>test data<span class=\'contentQuote\'>"</span></p>';
+        $aboutMeQuote = [['content' => 'test data']];
+        $case = displayAboutMeQuote($aboutMeQuote);
         $this->assertEquals($expected, $case);
     }
 
     public function testDisplayAboutMeQuoteFailure()
     {
         $expected = "";
-        $aboutMeText = [['thing'=>'words']];
-        $case = displayAboutMeText($aboutMeText);
+        $aboutMeQuote = [['thing'=>'words']];
+        $case = displayAboutMeQuote($aboutMeQuote);
         $this->assertEquals($expected, $case);
     }
 
     public function testDisplayAboutMeQuoteMalform()
     {
-        $aboutMeText = 'string';
+        $aboutMeQuote = 'string';
         $this->expectException(TypeError::class);
-        displayAboutMeText($aboutMeText);
+        displayAboutMeQuote($aboutMeQuote);
+    }
+
+    public function testCleanAboutMeTextSuccess()
+    {
+        $expected = "string";
+        $addAboutMeText = "  string  ";
+        $case = CleanAboutMeText($addAboutMeText);
+        $this->assertEquals($expected, $case);
+    }
+
+    public function testCheckAddMeTextSuccess() {
+        $expected = true;
+        $addAboutMeText = "";
+        $case = checkAddMeText($addAboutMeText);
+        $this->assertEquals($expected, $case);
+    }
+
+    public function testCheckAddMeTextFailure() {
+        $expected = false;
+        $addAboutMeText = "string";
+        $case = checkAddMeText($addAboutMeText);
+        $this->assertEquals($expected, $case);
+    }
+
+    public function testCheckAddMeTextMalform()
+    {
+        $input = ['thing'];
+        $this->expectException(TypeError::class);
+        checkAddMeText($input);
+    }
+
+    public function testAddAboutMeSuccessFalse() {
+        $expected = "<p>Please add content of up to 400 characters</p>";
```

Comment:
functions () {
}

tests () 
{

}

this is the format that james does so i copied it :)

Reply:
Done